lib/src/components/NxTransferListHalf/NxTransferListHalf.tsx

```diff
@@ -40,14 +40,17 @@ function _TransferListItem<T extends string | number = string>(props: TransferLi
 
   function onChange(evt: FormEvent<HTMLInputElement>) {
     // NOTE: the `checked` property on the DOM node will have the new value, not the old
-    onChangeProp(evt.currentTarget.checked, id);
+    onChangeProp?.(evt.currentTarget.checked, id);
   }
 
   const classes = classnames(
       'nx-transfer-list__item',
       {
         'nx-transfer-list__item--with-reordering': !!showReorderingButtons
       },
+      {
+        'nx-transfer-list__moving--disabled': !onChangeProp
```

Comment:
This isn't correct BEM.  I'd call it `nx-transfer-list__item--moving-disabled`.  Or put a `.nx-transfer-list__half--moving-disabled` class on the `.nx-transfer-list__half`

Reply:
> This isn't correct BEM. I'd call it nx-transfer-list__item--moving-disabled. Or put a .nx-transfer-list__half--moving-disabled class on the .nx-transfer-list__half

Better yet make a class for the opposite case: a `.nx-transfer-list__item--movable` or something.  Writing the CSS for that should be easier

Reply:
Ah I miss the item.
Thanks for the suggestions, updated here 6f2d800213933eb20a58d91c8b7f579d8ef176b8.